docs: reorient chatgpt sample `README` to favor cloud (#175)
autokitteh/samples: samples/openai_chatgpt/README.md

# OpenAI ChatGPT Sample

This project demonstrates integration with ChatGPT for text generation and response analysis. It showcases how to make API calls to ChatGPT and track token usage statistics.

API documentation:

- [OpenAI developer platform](https://platform.openai.com/)
- [Python client API](https://github.com/openai/openai-python)

> ## How It Works

Reviewer: Personally, I think this section in redundant in this case. There really isn't any notable "flow" or "algorithm" other than "get input, send to service, print output" - which is trivial and obvious.

Author: Done

1. Receive text input via HTTP POST requests
2. Send the input to ChatGPT's API for processing

> 3. Return the AI-generated response and token usage statistics

Reviewer: ```suggestion
3. Print the AI-generated response and token usage statistics
```

## Cloud Usage

> 1. Initialize your OpenAI connection

Reviewer: ```suggestion
1. Initialize your OpenAI ChatGPT connection
```

2. Copy the webhook trigger's URL:
   - Hover over the trigger's (i) icon
   - Click the copy icon next to the webhook URL

>    - (Detailed instructions [here](https://docs.autokitteh.com/get_started/deployment#webhook-urls))

Reviewer: I think this step is based on my previous comment somewhere, but I'm convinced now that it's too verbose. These are shorter and better:

https://github.com/search?q=repo%3Aautokitteh%2Fkittehub+%22from+the+%5C%22triggers%5C%22+tab%22&type=code

Author: Done

3. Deploy project

## Trigger Workflow

> [!IMPORTANT]

> > Ensure the OpenAI integration is initialized with a valid API key; otherwise, workflows will raise `ConnectionInitError`.

Reviewer: ```suggestion
> Ensure the OpenAI ChatGPT integration is initialized with a valid API key; otherwise, workflows will raise `ConnectionInitError`.
```

> Send an HTTP POST request to trigger the workflow:

Reviewer: ```suggestion
Send an HTTP POST request to the webhook URL from step 2 in the [Cloud Usage](#cloud-usage) section above:
```

```shell

> curl -X POST "http://localhost:9980/webhooks/<webhook_slug>" -H "Content-Type: text/plain" -d "Meow"

Reviewer: Don't assume a specific hostname, especially not a local one
```suggestion
curl -i -X POST "${WEBHOOK_URL}" -H "Content-Type: text/plain" -d "Meow"
```

```

> [!TIP]
> You can modify the request body to send custom text and observe ChatGPT's dynamic responses.

## Self-Hosted Deployment

> Follow these [detailed instructions](https://docs.autokitteh.com/get_started/deployment) to deploy the project on a self-hosted server.

Reviewer: ```suggestion
Follow [these detailed instructions](https://docs.autokitteh.com/get_started/deployment) to deploy the project on a self-hosted server.
```
